f8a_report/helpers/report_helper.py

```diff
@@ -411,15 +410,15 @@ def normalize_worker_data(self, start_date, end_date, stack_data, worker,
             'stacks_summary': {},
             'stacks_details': []
         }
-        all_deps = {'npm': [], 'maven': [], 'pypi': []}
-        all_unknown_deps = {'npm': [], 'maven': [], 'pypi': []}
+        all_deps = {'npm': [], 'maven': [], 'pypi': [], 'golang': []}
```

Comment:
Can we have a list of ecosystems defined and then build all these maps.

Reply:
Ack!